src/cpp/cpp_internal_additions.cpp

```diff
@@ -63,25 +63,28 @@ void cpp_internal_additions(std::ostream &out)
 
   // CPROVER extensions
   out << "const unsigned __CPROVER::constant_infinity_uint;" << '\n';
-  out << "typedef void __CPROVER_integer;" << '\n';
-  out << "typedef void __CPROVER_rational;" << '\n';
-  // TODO
-  // out << "thread_local unsigned long __CPROVER_thread_id = 0;" << '\n';
-  out << "__CPROVER_bool "
-      << "__CPROVER_threads_exited[__CPROVER::constant_infinity_uint];" << '\n';
-  out << "unsigned long __CPROVER_next_thread_id = 0;" << '\n';
+  out << "typedef void " CPROVER_PREFIX "integer;" << '\n';
+  out << "typedef void " CPROVER_PREFIX "rational;" << '\n';
+  // TODO: thread_local is still broken
+  // out << "thread_local unsigned long "
+  //     << CPROVER_PREFIX "thread_id = 0;" << '\n';
+  out << CPROVER_PREFIX "bool "
+      << CPROVER_PREFIX "threads_exited[__CPROVER::constant_infinity_uint];"
+      << '\n';
+  out << "unsigned long " CPROVER_PREFIX "next_thread_id = 0;" << '\n';
   out << "extern unsigned char "
-      << "__CPROVER_memory[__CPROVER::constant_infinity_uint];" << '\n';
+      << CPROVER_PREFIX "memory[__CPROVER::constant_infinity_uint];" << '\n';
```

Comment:
:pick: Looks like there are a few more `__CPROVER` here?  perhaps not, since not underscore after, maybe there should be ?

Reply:
This is the `__CPROVER` namespace, without an underscore at the end. Arguably it could be debated whether that should change as well, but it isn't literally the same as all the rest...